### PR TITLE
Document fundamental proof info

### PR DESCRIPTION
**Pull request by david-a-wheeler**
This documents fundamental (meta) proof information: description, variables, and disjoints.

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -1154,6 +1154,85 @@ The Editor tab lets you edit a proof; it starts empty.
...
+The "type" depends on the database, e.g., in set.mm it could be
+`class`, `wff`, or `setvar`.
+An example would be:
+
+~~~~metamath
+.loc1 class width
+~~~~
+
+Notice the ".". The period suppresses automatic deletion of 
+the line; without it, if the variable is unused then then entry will disappear.
```
Comment (expln) [inline docs/guide.md:1201]: typo `then then`

File: docs/guide.md [hunk 2]
```diff
@@ -1154,6 +1154,85 @@ The Editor tab lets you edit a proof; it starts empty.
...
+x,y,F
+x,y,u,v
+~~~~
+
+The disjoint expression `x,y` simply means that
+`x` and `y` must be disjoint variables (aka distinct variables).
+This means that they may not be simultaneously
+substituted with the same variable.
+The disjoint expression `x,ph` means
+variable `x` must not occur in the in wff `ph`.
```
Comment (expln) [inline docs/guide.md:1232]: typo `in the in wff` -> `in the wff`

File: docs/guide.md [hunk 3]
```diff
@@ -1154,6 +1154,85 @@ The Editor tab lets you edit a proof; it starts empty.
...
+
+For more information, see the Metamath book.
```
Comment (expln) [inline docs/guide.md:1234]: I think it would be nice to add a link to the metamath.org page with the Metamath book.

**Review by expln — COMMENTED** [review]
Thanks!

File: docs/guide.md (hunk 3 above)
Comment (david-a-wheeler) [reply docs/guide.md:1234]: Good idea. I'll do that as a fix to #62 .

**Pull request merged**